react-intro/react-intro-quiz-notes.md

```diff
@@ -0,0 +1,42 @@
+# react-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is React?
+  react is a JavaScript library for building user interfaces based on components.
+- What is a React component?
+  a piece of the UI that has it's own logic and apperence.
+- How do you mount a React app (root component) to the DOM?
+  npm create vite@latest (app-name) -- --template react
```

Comment:
Not quite. That's how we create react apps. How are we actually mounting it to our HTML file? Look at `App.jsx`